<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cono</class>
 <widget class="QDialog" name="cono">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cono</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 41, 41);</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>113</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Dibujar-Borrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflexion Vertical</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxXinicio">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="placeholderText">
    <string>Horizontal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Trasladar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflexion Horizontal</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxGrados">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="placeholderText">
    <string>Grados</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxAltura">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="placeholderText">
    <string>Altura</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxYinicio">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="placeholderText">
    <string>Vertical</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
